controller/node_controller.go

```diff
@@ -105,50 +103,45 @@ func NewNodeController(
 
 	nc.scheduler = scheduler.NewReplicaScheduler(ds)
 
-	// We want to check the real time usage of disk on nodes.
-	// Therefore, we add a small resync for the NodeInformer here
-	nodeInformer.Informer().AddEventHandlerWithResyncPeriod(cache.ResourceEventHandlerFuncs{
+	nodeInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
I think we could leave a comment to explain _why_ the node informer uses `AddEventHandler` rather than `AddEventHandlerWithResyncPeriod`.

Reply:
Agree. Added back the comment

Reply:
Actually, I think we dont need the comment because it is similar to other controller: the node controller will resync node CR every 30s

Reply:
I see, I just want to have a comment/history somewhere to explain why this one does not use `AddEventHandlerWithResyncPeriod`.

Reply:
All developers can get the details by checking the git annotation/commit comment.